projects/core/components/root/root.style.less

```diff
@@ -42,7 +42,7 @@ tui-root > .t-root-scrollbar {
     isolation: isolate;
 }
 
-// TODO: Required by Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1845264
 .t-overscroll-none {
     overscroll-behavior: none;
+    overflow: hidden;
```

Comment:
I thought it would reset the global scroll position 🤔

Reply:
`position: fixed` will reset, but I checked `overflow: hidden` does not reset